src/apps/competitions/tasks.py

```diff
@@ -744,3 +745,20 @@ def update_phase_statuses():
     competitions = Competition.objects.exclude(phases__in=Phase.objects.filter(is_final_phase=True, end__lt=now()))
     for comp in competitions:
         comp.update_phase_statuses()
+
+
+@app.task(queue='site-worker')
+def submission_status_cleanup():
+    submissions = Submission.objects.filter(status=Submission.RUNNING).exclude(has_children=True).select_related('phase')
```

Comment:
nitpick, we could do it like this in the future :D (remove `exclude()`) -- I also added `select_related` for `parent`
```suggestion
    submissions = Submission.objects.filter(status=Submission.RUNNING, has_children=False).select_related('phase', 'parent')
```

Reply:
Ah yeah I saw someone use that when I was looking for exclude, but it sounds like they do the same thing and one doesn't necessarily perform better. I do like the way that looks better though so I'm down.